sdk/communication/Azure.Communication.CallingServer/tests/Infrastructure/CallingServerTestEnvironment.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net;
+using Azure.Communication.Tests;
+
+namespace Azure.Communication.CallingServer.Tests
+{
+    /// <summary>
+    /// A helper class used to retrieve information to be used for tests.
+    /// </summary>
+    public class CallingServerTestEnvironment : CommunicationTestEnvironment
+    {
+        public string ToPhoneNumber => GetRecordedVariable(AzurePhoneNumber, options => options.IsSecret());
```

Comment:
We probably need a separate phone number for these tests to avoid conflicts with the phone number and sms packages tests